src/PageTournament/PageTourneyScores.res

```diff
@@ -167,7 +167,7 @@ module SelectTieBreaks = {
             {React.string(" Move down")}
           </button>
           <button
-            className={Cn.append("button-micro", "button-primary"->Cn.onSome(selectedTb))}
+            className={`button-micro ${selectedTb != None ? "button-primary" : ""}`}
```

Comment:
I replaced the `rescript-classname` with plain string interpolation

Reply:
That looks good to me.